node-tests/acceptance/build-test.js

```diff
@@ -73,10 +73,7 @@ describe('Acceptance', function() {
         expect(output.manifest()).to.deep.equal(expectedManifests['lazy']);
         output.contains('assets/node-asset-manifest.js');
         // Engine inherites its environment from the parent
-        output.contains(
-          `engines-dist/${engineName}/config/environment.js`,
-          /"environment":"development"/
-        );
+        output.contains(`engines-dist/${engineName}/config/environment.js`);
```

Comment:
I'll need to expand the tests, to support  multiple scenarios.

Reply:
cc: @stefanpenner